.github/scripts/fetch_external_sources.sh

```diff
@@ -8,36 +8,51 @@
 try_fetch() {
     local org=${1}
     local file=${2}
+    local outputbase=${3}
     local repo=$(echo ${file} | awk -F '/' '{print $1}')
     local repo_file=${file/${repo}/}
 
     for branch in main master; do
-      curl --fail --silent https://raw.githubusercontent.com/${org}/${repo}/${branch}/${repo_file#/} -o ${file} && break
+      curl --fail --silent https://raw.githubusercontent.com/${org}/${repo}/${branch}/${repo_file#/} -o ${outputbase}/${file} && break
     done
```

Comment:
If no argument is passed this is now `/${file}` which may not be fine if the workflow is not running in `/` (?) I don't remember which is the default for github workflows